CryptoTechReminderSystem/Gateway/HarvestGateway.cs

```diff
@@ -112,13 +112,12 @@ private JObject RetrieveFromEndPoint(string address)
 
         private async Task<JObject> GetApiResponse(string address)
         {   
-            int THROTTLE_TIME_IN_MS = 150;
-            int MAX_ATTEMPTS = 5;
-            
             var harvestRetryPolicy = Policy
                 .HandleResult<HttpResponseMessage>(r => r.StatusCode == System.Net.HttpStatusCode.TooManyRequests)
-                .WaitAndRetryAsync(
-                    MAX_ATTEMPTS, attempt => TimeSpan.FromMilliseconds(THROTTLE_TIME_IN_MS * Math.Pow(2, attempt)));
+                .WaitAndRetryAsync(new[]
+                {
+                    TimeSpan.FromSeconds(15),
+                });
```

Comment:
Thanks for the feedback @dushan-madetech and @StevenLeighton21! 😀  What do you think of this retry policy?

> "Retry once after 15 seconds" is chosen because a) the use case
> does not require shorter retry intervals and b) 15 seconds is the
> worst case before the API rate limit is reset.
> 
> Rate limit in Harvest API docs: https://tinyurl.com/y57vafxx

Reply:
LGTM